### ISSUE TITLE
Remote vehicle control partly stops working sometimes

### ISSUE THREAD

**Issue by sumpfralle** [CleverRaven/Cataclysm-DDA#46413]
# Describe the bug

I am using a remote vehicle controller for a farming vehicle. Usually everything works fine (speed, steering, electronics), while the vehicle is in range.

At one point I suddenly noticed, that I could not enter the `vehicle controls` menu anymore (hot key `^`, log message: `no vehicle controls found`). Only steering and speed control continued to work. I was able to stop controlling the vehicle by turning the remote controller off. After enabling the remote controller again, the vehicle controls menu (`^`) was still not reachable. Steering and speed control worked (as before).

The only way to adjust electronics (e.g. enable/disable the plow) was to move to the vehicle and examine the control tile of the vehicle manually.

# Steps To Reproduce

I saved the game (and quit it) at the moment, when the `no vehicle controls found` message was emitted. But after loading the game again, the vehicle controls menu was reachable again. Thus loading a game probably resets the broken state variable.

# Expected behavior

The vehicle controls menu (`^`) should always be reachable, while I am driving a vehicle.

# Versions and configuration

 - OS: Debian testing
 - Game Version: 0.E-8544-g0c9c8e7
 - Graphics version: tiles
 - Ingame language: English
 - Mods loaded: default

**Comment by int-ua** [comment]
I'm afraid we may need a better steps to reproduce.

**Comment by sumpfralle** [comment]
Thanks for taking a look at my report!

Meanwhile I noticed, that the vehicle controls menu (`^`) does not work with a remote controller anymore (not even right after loading a game).
The electronic components (e.g. the *plow* or *reaper*) can now be controlled by *using* the remote controller and picking *execute single action*.
Thus maybe the previous approach (using the `^` menu) was replaced by this new interface approach.
In this case: it feels a bit more cumbersome, but I guess, the change was done for a good reason.

Thus I assume, that my experience (randomly losing access to the `^` menu for a remote controlled vehicle) is not relevant anymore, since the interface was changed recently.
